pkg/argocd/argocd_test.go

```diff
@@ -188,7 +188,7 @@ func Test_FilterApplicationsForUpdate(t *testing.T) {
 				},
 				Spec: v1alpha1.ApplicationSpec{},
 				Status: v1alpha1.ApplicationStatus{
-					SourceType: v1alpha1.ApplicationSourceTypeKsonnet,
+					SourceType: v1alpha1.ApplicationSourceTypePlugin,
```

Comment:
Ksonnet was deprecated and removed from Argo CD, so we need to use a different unsupported source type here.